components/chart/base/MultiBarBaseChart.tsx

```
@@ -0,0 +1,101 @@
+                return Math.max(...value.map((value2, index2) => {
+                    return value2 + valuesOffset?.[index]?.[index2]! }))
+            })),
+        [ values, valuesOffset ],
+    );
+
+    const flatValues: number[] = useMemo(
+        () => {
+            // Sum values together with the offset
+            const valuesWithOffset = [...values.map((value, index) => {
+                return [...value.map((value2, index2) => {
+                    return value2 + valuesOffset?.[index]?.[index2]! })]
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)